src/Libraries/RevitNodes/Elements/FillPatternElement.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Autodesk.DesignScript.Runtime;
```

Comment:
Fill pattern is not part of the plan but does not do harm I guess.. Just more testing effort~